build.gradle

```diff
@@ -15,3 +15,9 @@ allprojects {
     apply from: rootProject.file("build-tools/repositories.gradle")
     apply plugin: "aditogether.lint"
 }
+
+subprojects {
+    tasks.withType(Test).configureEach {
+        useJUnitPlatform()
+    }
+}
```

Comment:
This seems to me quick a big assumption to have in the root project.
The optimal setup would be IMHO something like:
"if kotest is applied, than use junit platform"

But before applying such behavior, I think it would be better to have some real examples on our targets and move this in a shared config only after we see how it's used.

What do you think?

Reply:
Yes, I agree, I had mixed feelings myself